scripts/newhistoryimport.sql

```diff
@@ -0,0 +1,79 @@
+# One time script for import of new database history tables provided by Jeremy circa Jan 2022.
+#
+# Note: This script will drop all of the hist_ tables that are generated by the import, so it
+#       CAN ONLY BE USED FOR FULL IMPORTS.  This includes both the hist_new_raceentries temp
+#       table and the final generated hist_raceentries / hist_entrypeoplemap tables.
+#
+# Steps for import:
+# - Import the history data provided by Jeremy
+# - Run this script to rename tables to hist_ format.
+#   - Additionally, this script adds indexes that appear to be useful but were not in the original
+#     table definitions.
+# - Run second script to denormalize hist_new_raceentries table into hist_raceentries and hist_entrypeoplemap
+# - If all is well, (manually) drop hist_new_raceentries.
```

Comment:
Is the idea to keep it around for manual review, then drop it?

Reply:
Yeah, mostly to sanity check -- I envision this being like, "check site works with new data, then drop the temp table".

If we're confident, we can just always drop it at the end of the normalize script too.